src/ds5/ds5-fw-update-device.cpp

```diff
@@ -19,6 +19,17 @@ namespace librealsense
         update_device::update(fw_image, fw_image_size, callback);
     }
 
+    bool ds_update_device::check_fw_compatibility(const std::vector<uint8_t>& image) const
+    {
+        std::string fw_version = extract_firmware_version_string((const void*)image.data(), image.size());
+        auto device_pid = _usb_device->get_info().pid;
+        auto it = ds::device_to_fw_min_version.find(_usb_device->get_info().pid);
+        if (it == ds::device_to_fw_min_version.end())
```

Comment:
In case the upper FW version limit is set, the check should check both the lower and upper bounds

Reply:
There is no upper FW version limit map for ds5